requirements.txt

```diff
@@ -0,0 +1,13 @@
+ansicolors==1.1.8
+boto3==1.5.32
+botocore==1.8.50
+docutils==0.16
+jmespath==0.9.3
+python-dateutil==2.8.1
+PyYAML==4.2b4
+s3transfer==0.1.13
```

Comment:
I don't think I ever directly use this library. I think it is a requirement for botocore maybe and gets automatically installed.

Reply:
Yes, those are automatically installed. I generated this from running pip install for the exact versions mentioned in `setup.py`, then `pip freeze` and putting the exact ones needed in requirements.txt